src/main/java/lab5/players/Omola.java

```diff
@@ -0,0 +1,40 @@
+package lab5.player;
+
+import lab5.Board;
+import lab5.PlayerToken;
+import java.util.List;
+
+public class Omola extends Player {
+
+    public Omola(String name) {
+        super(name);
+    }
+
+    @Override
+    public int pickNextMove(Board board) {
+        List<Integer> emptyCells = board.getEmptyCells();
+        PlayerToken myToken = board.getNextTurnToken();
+
+        // Check for a winning move
+        for (int move : emptyCells) {
+            Board testBoard = new Board(board);
+            testBoard.placeNextToken(move);
+            if (testBoard.getWinner() == myToken) {
+                return move;
+            }
+        }
+
+        // Check for a blocking move
+        PlayerToken opponent = myToken.opponent();
+        for (int move : emptyCells) {
+            Board testBoard = new Board(board);
+            testBoard.placeNextToken(move);
+            if (testBoard.getWinner() == opponent) {
+                return move;
+            }
+        }
```

Comment:
Whoops!  Again, pay attention to your AI-generations... there's a bug here: you need to `place(move, opponent)` here instead of `placeNextMove` because you want to check what happens if the *opponent* moves here, not the current player.